Source/Core/Core/PowerPC/MMU.h

```diff
@@ -310,7 +310,9 @@ class MMU
   template <XCheckTLBFlag flag, bool never_translate = false>
   void WriteToHardware(u32 em_address, const u32 data, const u32 size);
   template <XCheckTLBFlag flag>
-  bool IsRAMAddress(u32 address, bool translate);
+  bool IsTranslatedRAMAddress(u32 address);
+  template <XCheckTLBFlag flag>
+  bool IsRAMAddress(u32 address) const;
```

Comment:
What these functions do would be made clearer by using the proper PowerPC terminology:

```suggestion
  bool IsEffectiveRAMAddress(u32 address);
  template <XCheckTLBFlag flag>
  bool IsPhysicalRAMAddress(u32 address) const;
```